### ISSUE TITLE
Dockless mobility operator area web map

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#1676]
Taking the data and converting the screenshots for the Current Operators areas seen on https://austintexas.gov/docklessmobility into a web map app on ArcGIS Online. The plan is to display the areas of the operators with a supplemental license and also show the Downtown Austin Project Coordination Zone (DAPCZ), which is the main area of all dockless operators. Also to be included on the map is a point feature for the dockless scooter and bike parking areas.

*Migrated from [atd-geospatial #20](https://github.com/cityofaustin/atd-geospatial/issues/20)*

**Comment by atdservicebot** [comment]
*From mstevens-atx on 2019-05-06*:

Finished initial design and creation of the dockless service area web map for now. It can be found [here](http://austin.maps.arcgis.com/apps/webappviewer/index.html?id=772a1ae0733e42f5a568b7d6e0288be1). At the time of this comment the map needs a CoA AGOL account login to view as it is not yet open to the public. 

Waiting on the DAPCZ to do the MAINT->DM->AGOL jump as that is the boundary of the majority of dockless providers at this time.

**Comment by atdservicebot** [comment]
*From mstevens-atx on 2019-05-20*:

The DAPCZ feature was added to AGOL and subsequently added to the map. Currently it is only shared with the organization and can be found here: http://austin.maps.arcgis.com/apps/webappviewer/index.html?id=772a1ae0733e42f5a568b7d6e0288be1

I added a splash page to try to get some more information in the map, but it needs some help both visually and with the content.

**Comment by atdservicebot** [comment]
*From mstevens-atx on 2019-05-31*:

Moving this to Icebox to put it on hold for now. John mentioned in a 1:1 needing to talk with Shannon to get some data things worked out with the different operators.

**Comment by atdservicebot** [comment]
*From mstevens-atx on 2019-07-03*:

The files and mxd for this can be found in G:\ATD\ATD_GIS\Parking_Enf_Mgmt\103480_Dockless_Area_of_Operations. There has been no change to the AGOL map since the last comment.

**Comment by JaceDeloney** [comment]
This is almost two years old. Closing out. This has not come up as a need from the SM team
